mariadb/10.5.11/README.md

```diff
@@ -0,0 +1,34 @@
+# Overview
+
+MariaDB comes with a bundled version of wolfSSL that can be used with the MariaDB server. This is limited in a few ways:
+
++ libmariadb, which is used by MariaDB clients, isn't set up to be built with wolfSSL. If the server is built with wolfSSL, libmariadb gets builts with GnuTLS.
++ The wolfSSL version is tied to whatever version MariaDB has been bundled with.
++ The bundled version isn't FIPS.
+
+This port makes it so a system-installed version of wolfSSL can be used for both the server and libmariadb. This also makes it possible to use wolfSSL FIPS with MariaDB. This has been tested with both wolfSSL FIPS and non-FIPS 4.8.1.
+
+# Build Instructions
+
+## Build wolfSSL
++ Configure wolfSSL with `./configure CPPFLAGS="-DRSA_MAX_SIZE=8196 -DFP_MAX_BITS=16384 -DNO_OLD_TIMEVAL_NAME -DWOLFSSL_MYSQL_COMPATIBLE -DWOLFSSL_AES_DIRECT -DKEEP_OUR_CERT -DHAVE_AES_ECB -DWOLFSSL_NO_DEF_TICKET_ENC_CB -DHAVE_EX_DATA" --enable-opensslall --enable-crl --enable-sessioncerts --enable-aesctr --enable-rsapss --enable-keygen --enable-des3`. Add `--enable-debug` if you want to enable the debug version of wolfSSL.
++ Compile with `make`.
++ Install wolfSSL into /usr/local with `sudo make install`.
+
+## Build MariaDB
++ Clone the MariaDB server GitHub repo with `git clone git@github.com:MariaDB/server.git mariadb-server`. `cd mariadb-server`.
++ Check out the 10.5.11 tag with `git checkout mariadb-10.5.11`.
++ Set up the libmariadb submodule with `git submodule init && git submodule update`.
++ Patch the source code with `patch -p1 < mariadb-10.5.11.patch`, adjusting the path to the patch file accordingly. 
++ Create a build directory with `mkdir build` and `cd build`.
++ Configure ntp with `cmake -DWITH_SSL=system-wolfssl ..`. Add `-DCMAKE_BUILD_TYPE=Debug` if debugging. Note that this configuration requires libwolfssl and the wolfSSL headers to be installed in a standard location that can be found by CMake (e.g. /usr/local).
```

Comment:
Seems like "Configure ntp" should be "Configure mariadb" here?

Reply:
Fixed.